cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -314,7 +316,7 @@ jobs:
     steps:
       # Get all the artifacts for the signing tasks to use
       - name: Fetch local artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: artifacts
           path: target/distrib/
```

Comment:
missing merge-multiple/pattern (let's just be safe here).

Reply:
Yeah, I left this out because this is the Windows signing job I was asking about